elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -35,7 +35,7 @@ trait QueryDsl extends DslCommons with SortDsl {
   case class QueryRoot(query: Query,
                        fromOpt: Option[Int],
                        sizeOpt: Option[Int],
-                       sort: Seq[Sort],
+                       sortOpt: Option[Seq[Sort]],
```

Comment:
Sure.  This was actually changed from Option to non-option in revision 9f258d3 (8/22/2016) so I thought it would be OK to switch it back.
If you still want me to do it to change it back, how do I update the pull request?  Sorry, I don't do this often.